README.md

```diff
@@ -33,6 +33,83 @@ The plan is to use OCI projects and best of breed libraries for different aspect
 
 It is currently in active development in the Kubernetes community through the [design proposal](https://github.com/kubernetes/kubernetes/pull/26788).  Questions and issues should be raised in the Kubernetes [sig-node Slack channel](https://kubernetes.slack.com/archives/sig-node).
 
+## Getting started
+
+### Prerequisites
+`runc` is expected to be installed and be available in the PATH.
+
+Build
+
+```
+GOPATH=/path/to/gopath
+mkdir $GOPATH
+go get github.com/kubernetes-incubator/cri-o
```

Comment:
Do you need -d in go get to not try and install it? (ignore this if I'm misremembering) a